.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,55 @@
+name: Run tests
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6, 3.7]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        python -m pip install --upgrade pip
+        hash -r
+        pip3 install -r pip-dep/requirements.txt
+        pip3 install -r pip-dep/requirements_dev.txt
+        pip3 install -r pip-dep/requirements_udacity.txt
+        pip3 install -r pip-dep/requirements_notebooks.txt
+        pip3 install scikit-learn
+        python setup.py install
+    - name: Lint, format, and type-check
+      run: |
+        pip3 install flake8
+        #- pip3 install flake8-docstrings
+        pip3 install flake8-comprehensions
+        pip3 install pep8-naming
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 --config=.flake8 .
+        black --check --verbose .
+        # exit-zero treats all errors as warnings.
+        # diverting from the standard 79 character line length in
+        # accordance with this:
+        # https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds
+        flake8 . --count --exit-zero --statistics --select=E,F,W,C90
+        # Run type checker in CI such that type errors can be identified and gradually addressed.
+        # Once all existing type errors are addressed, the `|| echo` after the mypy call can be removed.
+        mypy syft || echo "Type errors found, continuing build..."
+    - name: Test with pytest
+      run: |
+        coverage run -m pytest test --deselect=test/notebooks/test_notebooks.py::test_notebooks_basic_translations --deselect=test/notebooks/test_notebooks.py::test_all_translation_notebooks --deselect=test/efficiency
+    - name: Coverage reports
+      run: |
+        coverage report --omit="syft/grid/*" --ignore-errors --fail-under 95 -m
```

Comment:
`--omit="syft/grid/*"` is removed in travis config, but left here - is that intentionally?

Reply:
Ah, forgot to remove it here. It’s been moved into setup.cfg.